dspace-api/src/test/java/org/dspace/app/requestitem/RequestItemEmailNotifierTest.java

```diff
@@ -171,6 +197,160 @@ public void testSendResponse() throws Exception {
                 (String)content, containsString(TEST_MESSAGE));
     }
 
+    /**
+     * Test of sendResponse method with attached files, of class RequestItemEmailNotifier.
+     * @throws java.lang.Exception passed through.
+     */
+    @Test
+    public void testSendResponseWithFiles() throws Exception {
+        // Create some content to send.
+        context.turnOffAuthorisationSystem();
+        Community com = CommunityBuilder.createCommunity(context).withName("Top Community").build();
+        Collection col = CollectionBuilder.createCollection(context, com).build();
+        Item item = ItemBuilder.createItem(context, col).withTitle("Test Item").build();
+
+        Group group = GroupBuilder.createGroup(context).withName("Group").build();
+        Group anonymousGroup = EPersonServiceFactory.getInstance().getGroupService().findByName(context, ANONYMOUS);
+        EPerson eperson = EPersonBuilder.createEPerson(context).withEmail("test@test.test").build();
+
+        Bundle bundle = BundleBuilder.createBundle(context, item).withName("ORIGINAL").build();
+
+        Bitstream bitstreamWithoutPolicies = createBitstream(bundle, "Bitstream without policies");
```

Comment:
there is a `org.dspace.builder.BitstreamBuilder` that should be used there or do you hit any issue using it?

Reply:
If you look at the `createBitstream()` method you can see that it uses `org.dspace.builder.BitstreamBuilder`